### PR TITLE
WSTEAM1-465: Update Live Page to pass a page number 

### PR DESCRIPTION
**Pull request by alex-magana**
Resolves JIRA  [WSTEAM1-465](https://jira.dev.bbc.co.uk/browse/WSTEAM1-465)

Overall changes
======
Update page number support for the Live Page.

Code changes
======

- Set the Live Page to render page 1 as a default.
- Serve a 404 for page requests below 1 and above 50.

Testing
======
1. _List the steps used to test this PR._

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ import getAgent from '../../../../utilities/undiciAgent';
 
 import LivePageLayout from './LivePageLayout';
 import extractHeaders from '../../../../../src/server/utilities/extractHeaders';
+import isValidPageNumber from '../../../../utilities/pageQueryValidator';
```
Comment (HarveyPeachey) [inline ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx:23]: Is this logic necessary if the request is filtered out downstream via Belfrage?

Comment (alex-magana) [reply ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx:23]: Thanks for the feedback, Harvey.
I'd venture that it is. The rationale behind this being that validation is performed
independent of the validation other services are performing.
The other reason for this check, is to keep the app from firing off
requests to the BFF for invalid page numbers.
Happy to remove the check if it's considered redundant.

Comment (HarveyPeachey) [reply ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx:23]: Discussed in call, LGTM

**Review by HarveyPeachey — APPROVED** [review]

File: ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx [hunk 2]
```diff
@@ -96,11 +97,26 @@ export const getServerSideProps: GetServerSideProps = async context => {
     service,
     variant,
     // renderer_env: rendererEnv,
-    page,
+    page = '1',
   } = context.query as PageDataParams;
 
   const { headers: reqHeaders } = context.req;
 
+  if (!isValidPageNumber(page)) {
```
Comment (amoore108) [inline ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx:105]: Will this error if no page number is given? I feel like the URL on the initial request won't have the `page` query param, e.g.

Initial request: `pidgin/live/xxxx`
Clicked pagination: `pidgin/live/xxxx?page=2`
Clicked back to page 1: `pidgin/live/xxxx?page=1`

Comment (amoore108) [reply ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx:105]: Actually ignore! I missed the `page` getting set to 1 by default in the `context.params` destruct 🙃

**Review by amoore108 — APPROVED** [review]
Nice 👍

**Pull request merged**